erigon-lib/common/cmp/cmp.go

```diff
@@ -17,11 +17,11 @@
 package cmp
 
 import (
-	"golang.org/x/exp/constraints"
+	"cmp"
 )
 
 // InRange - ensure val is in [min,max] range
-func InRange[T constraints.Ordered](min, max, val T) T {
+func InRange[T cmp.Ordered](min, max, val T) T {
```

Comment:
I guess we could move this to `generics.go` and get rid of this file altogether.

Reply:
the `generics` package is meant for generics related utilities, think of them as generics handy tools
`InRange` is a comparison utility, so I do not think it should be part of `generics` because although it uses generics it is not a generics related utility